services/storage/.env-devel

```diff
@@ -23,6 +23,7 @@ S3_BUCKET_NAME=simcore
 # 172.17.0.1 is the docker0 interface, which redirect from inside a container onto the host network interface.
 S3_ENDPOINT=http://172.17.0.1:9001
 S3_SECRET_KEY=12345678
+S3_REGION=us-east-1
```

Comment:
THOUGHT: having dot-env files per service and globally (i.e. for the docker-compose specs) is redundant. This is the result of two different strategies that never converged.  These dot-env per service are only used for testing or local development. E.g. in `mock_environment` fixture for testing, or to start locally the service (probably not done anymore)

We could go two ways:
1. have a dot-env per service and merge them into one for the compose specs (probably requires also a `common` listing for env vars shared)
2. have a global dot-env and every service parses what it is needed.

The second approach seems more feasible for me.

Reply:
also prefer second option.